pkg/option/config.go

```diff
@@ -43,6 +43,8 @@ type config struct {
 	EventQueueSize uint
 
 	ReleasePinned bool
+
+	EnablePolicyFilter bool
```

Comment:
Maybe a comment here to... that this will be removed

Reply:
I'm not sure if we are going to remove it. First, we will default it to `true`.